src/lib/openapi/spec/me-schema.ts

```diff
@@ -0,0 +1,43 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { userSchema } from './user-schema';
+import { permissionSchema } from './permission-schema';
+import { feedbackSchema } from './feedback-schema';
+
+export const meSchema = {
+    $id: '#/components/schemas/meSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['user', 'permissions', 'feedback', 'splash'],
```

Comment:
Q: is it intuitive that "array" types are required? I assume they can be an empty list.